graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -124,14 +124,14 @@ def _get_fields_for_class(schema_graph, graphql_types, field_type_overrides, hid
     schema_element = schema_graph.get_element_by_class_name(cls_name)
     outbound_edges = (
         ('out_{}'.format(out_edge_name),
-         schema_graph.get_element_by_class_name(out_edge_name).properties[
-             EDGE_DESTINATION_PROPERTY_NAME].qualifier)
+         get_only_element_from_collection(schema_graph.get_element_by_class_name(
```

Comment:
I disagree with this change. I think `in_connections` and `out_connections` should contain _all_ types that a given type could be connected to, including the subclasses of the type specified by the schema.